R/wwinference.R

```diff
@@ -350,42 +369,49 @@ fit_model <- function(compiled_model,
 #' This function returns a list of MCMC settings to pass to the
 #' `cmdstanr::sample()` function to fit the model. The default settings are
 #' specified for production-level runs, consider adjusting to optimize
-#' for speed while iterating.
+#' for speed while iterating. All input arguments to `cmdstanr::sample()`
+#' are configurable by the user. See `cmdstanr::sample()` documentation
+#' https://mc-stan.org/cmdstanr/reference/model-method-sample.html for a full
+#' description of each argument.
 #'
 #'
 #' @param iter_warmup integer indicating the number of warm-up iterations,
 #' default is `750`
 #' @param iter_sampling integer indicating the number of sampling iterations,
 #' default is `500`
-#' @param n_chains integer indicating the number of MCMC chains to run, default
+#' @param chains integer indicating the number of MCMC chains to run, default
 #' is `4`
-#' @param seed set of integers indicating the random seed of the stan sampler,
+#' @param parallel_chains integer indicating the number of chains to run
+#' in parallel, default is `4`
+#' @param seed set of integers indicating the random seed of the Stan sampler,
 #' default is NULL
 #' @param adapt_delta float between 0 and 1 indicating the average acceptance
 #' probability, default is `0.95`
 #' @param max_treedepth integer indicating the maximum tree depth of the
 #' sampler, default is 12
+#' @param show_messages logical indicating whether to print all output
+#' during the execution process, default is `TRUE`
 #'
-#' @return a list of mcmc settings with the values given by the  function
+#' @return a list of MCMC settings with the values given by the  function
 #' arguments
-#' @export
 #'
-#' @examples
-#' mcmc_settings <- get_mcmc_options()
 get_mcmc_options <- function(
     iter_warmup = 750,
     iter_sampling = 500,
-    n_chains = 4,
+    chains = 4,
+    parallel_chains = 4,
     seed = NULL,
     adapt_delta = 0.95,
-    max_treedepth = 12) {
+    max_treedepth = 12,
+    show_messages = TRUE) {
```

Comment:
I think I had specified it because I thought this would be a useful and frequent one to modify, but since that is no longer the purpose of this function I will remove!